docs/install-kubernetes.md

```diff
@@ -97,13 +97,18 @@ Note the reference to the pre-created `registry-credentials` secret in `global.r
 helm --namespace mlrun \
     install mlrun-kit \
     --wait \
+    --timeout 960 \
     --set global.registry.url=<registry-url> \
     --set global.registry.secretName=registry-credentials \
     v3io-stable/mlrun-kit
 ```
 
-Where `<registry-url` is the registry URL which can be authenticated by the `registry-credentials` secret (e.g., `index.docker.io/<your-username>` for Docker Hub>).
+Where `<registry-url>` is the registry URL which can be authenticated by the `registry-credentials` secret (e.g., `index.docker.io/<your-username>` for Docker Hub).
 
+```{admonition} Note
+First-time MLRun users will experience a relatively longer installation time because all required images 
+are being pulled locally for the first time(it will take an average of 10-15 minutes mostly depends on 
+your internet speed).```
```

Comment:
I think we should place the note for first-time users before the first helm install so that even if users are not running on minikube will understand it applies to them as well.